DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateApprovals/DelegateRejectionPage.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@inject IConfiguration Configuration
+
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@using Microsoft.Extensions.Configuration
+@model RejectDelegateViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/shared/cardWithButtons.css")" asp-append-version="true">
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegateApprovals.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Reject delegate registration";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["CurrentSystemBaseUrl"]}/tracking/dashboard";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Reject delegate registration</h1>
+    <h1 class="nhsuk-body-l">Are you sure you want to reject this user's registration?</h1>

```

Comment:
I don't think this should be in a separate h1 element, but I'm not sure. Is there a reason why it is rather than being in a \<p> or \<h2> etc.?